### PR TITLE
fix(learn): Remove errors discovered while following Angular todo list app Guide

### PR DESCRIPTION
**Pull request by Pignated**
Imported CommonModule (could only import NgFor instead) and set app component as standalone. Without importing NgFor, ngForOf won't bind to li used in the template. Angular expects the component to be standalone. There could be better ways to fix this, I'm writing this while going through the tutorial.

### Description

Imported CommonModule and added standalone flag to app.components.ts. Otherwise the built app does not work as intended (or at all in the case of the standalone flag)

### Motivation
Without these changes (or others, I am not the most knowledgeable) I was unable to proceed with the tutorial. There could be better fixes for the issues I encountered, but following the current instructions exactly does not work.

### Related issues and pull requests
I have not created an issue but upon my search of the word "angular" in issues and PRs there isn't anything about this problem.

### CODE REVIEW

**Comment by Pignated** [comment]
I did follow all steps, including the ones on the page before to set up angular. I am guessing the problem is that it was written for an older version

**Comment by hamishwillee** [comment]
Thanks. I'm unassigning myself as I know nothing about Angular.

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/angular_todo_list_beginning/index.md [hunk 1]
```diff
@@ -92,8 +92,11 @@ In `app.component.ts`, replace the contents with the following:
 
 ```js
 import { Component } from "@angular/core";
+import { CommonModule } from "@angular/common";
 
 @Component({
+  standalone:true,
+  imports:[CommonModule],
```
Comment (OnkarRuikar) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/angular_todo_list_beginning/index.md:99]: Just a nit:
```suggestion
  standalone: true,
  imports: [CommonModule],
```

**Review by bsmth — APPROVED** [review]
Thanks a lot, this does fix the first set of errors, there are a couple of smaller fixes to do as a follow-up, I'll take a look at that separately 👍🏻

**Pull request merged**